.github/workflows/pr_release.yaml

```diff
@@ -0,0 +1,139 @@
+name: Build images on pull requests
+
+on: [pull_request]
```

Comment:
secrets are not available to `pull_request` but they are available to `pull_request_target` (the script is executed from the base of the PR though)

Reply:
yep. I read the fine print 4 hours late :(